### ISSUE TITLE
Fix installation instruction for glog dependency

### ISSUE THREAD

**Issue by vinothkr** [BVLC/caffe#5192]
Manual installation instruction of `glog` dependency [here](http://caffe.berkeleyvision.org/install_apt.html) fails because  [this](https://google-glog.googlecode.com/files/glog-0.3.3.tar.gz) doesn't exist anymore. 

Can we update this to point to the github itself?

**Comment by shelhamer** [comment]
Yes, the link should be fixed. It looks like this might work: https://github.com/google/glog/archive/v0.3.4.tar.gz
